web/pages/[username]/[contractSlug].tsx

```diff
@@ -276,23 +272,9 @@ export function ContractPageContent(
           userBets={userBets}
           comments={comments}
         />
-
-        {!user ? (
-          <Col className="mt-4 max-w-sm items-center xl:hidden">
-            <BetSignUpPrompt />
-            <PlayMoneyDisclaimer />
```

Comment:
Does the PlayMoneyDisclaimer otherwise appear on mobile for signed out users? I think we could relocate it, but not remove it.